sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -520,6 +520,7 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
             "TOK_TBLTEXTFILE", // Stored as TextFile
             "TOK_TBLRCFILE", // Stored as RCFile
             "TOK_TBLORCFILE", // Stored as ORC File
+            "TOK_TBLPARQUETFILE", // Stored as PARQUET
```

Comment:
For these tokens, only `TOK_TABNAME`, `TOK_QUERY`, and `TOK_IFNOTEXISTS` are actually used by `HiveQl`. Since we ask Hive's SemanticAnalyzer to create the `CreateTableDesc`, we basically ignore other tokens at here.